config.yaml

```diff
@@ -0,0 +1,13 @@
+kraken_db:
```

Comment:
Maybe differentiate `kraken` and `kraken2`? Their database formats are different.

Reply:
Done.